cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -78,11 +80,8 @@ def counts_to_results(
 
     samples: List[List[int]] = []
     for key in counter.keys():
-        keys_as_list: List[int] = []
-
         # Combines the keys of the counter into a list. If key = "01", keys_as_list = [0, 1]
-        for index in key:
-            keys_as_list.append(int(index))
+        keys_as_list: List[int] = list(map(int, key))
```

Comment:
might make sense to use `int(round(key))` to make this a little bit less wrong for ibm devices

Reply:
I'm not sure what you're suggesting here, `round` doesn't work on strings. It also already returns an int.

Reply:
oops, referenced the wrong thing in my suggestion - i just meant [this](https://github.com/Infleqtion/client-superstaq/pull/783#discussion_r1331974544) :)

Reply:
> oops, referenced the wrong thing in my suggestion - i just meant https://github.com/Infleqtion/client-superstaq/pull/783#discussion_r1331974544 :)

Done. (Yup, `round` returns an `int`.)